.eslintrc

```diff
@@ -5,10 +5,5 @@
   "parser": "babel-eslint",
   "rules": {
     "no-underscore-dangle": 0
-  },
-  "settings": {
-    "react": {
-      "pragma": "h"
```

Comment:
yep, deprecated and moved to `frint-compat`. just use React directly.

docs: https://github.com/Travix-International/frint/tree/next-monorepo/packages/frint-react#usage